lib/ansible/parsing/dataloader.py

```diff
@@ -53,23 +53,22 @@ class DataLoader:
         ds = dl.load_from_file('/path/to/file')
     """
 
-    def __init__(self):
+    def __init__(self) -> None:
 
-        self._basedir = '.'
+        self._basedir: str = '.'
 
         # NOTE: not effective with forks as the main copy does not get updated.
         # avoids rereading files
-        self._FILE_CACHE = dict()
+        self._FILE_CACHE: dict[str, t.Any] = {}
```

Comment:
Would `object` work here?
```suggestion
        self._FILE_CACHE: dict[str, object] = {}
```